lib/signet/oauth_2/client.rb

```diff
@@ -25,6 +25,7 @@ module Signet
   module OAuth2
     class Client
       OOB_MODES = ["urn:ietf:wg:oauth:2.0:oob:auto", "urn:ietf:wg:oauth:2.0:oob", "oob"].freeze
+      ID_TOKEN_TYPE = 'JWT'.freeze
```

Comment:
You don't seem to use this constant here or in g-a-l-r?

Reply:
It gets used in the `IapAccessToken` client middleware: https://github.com/LiveRamp/customerstorage/pull/28/files#diff-6fa7916f09ac6ea6ec95fc2819bf655cR51

There's probably some stuff in that `IapAccessToken` class that rightfully belongs in g-a-l-r, but that can be done in aa future push to get our fork merged upstream.